tests/test_rio_merge.py

```diff
@@ -540,3 +541,49 @@ def test_merge_precision(tmpdir, precision):
     result = runner.invoke(main_group, ["merge", "-f", "AAIGrid"] + precision + inputs + [outputname])
     assert result.exit_code == 0
     assert open(outputname).read() == textwrap.dedent(expected)
+
+
+@fixture(scope='function')
+def test_data_dir_resampling(tmpdir):
+    kwargs = {
+        "crs": {'init': 'epsg:4326'},
+        "transform": affine.Affine(0.2, 0, 0,
+                                   0, -0.2, 0),
+        "count": 1,
+        "dtype": rasterio.uint8,
+        "driver": "GTiff",
+        "width": 9,
+        "height": 1,
+        "nodata": 1
+    }
+
+    with rasterio.open(str(tmpdir.join('a.tif')), 'w', **kwargs) as dst:
+        data = np.ones((1, 9), dtype=rasterio.uint8)
+        data[:, :3] = 100
+        data[:, 3:6] = 255
+        dst.write(data, indexes=1)
+
+    return tmpdir
+
+
+@pytest.mark.parametrize("resampling", [resamp for resamp in Resampling if resamp < 7])
```

Comment:
All resampling methods from 0 to 7 included (gauss) should work correctly with `rio merge`, but for some reason with the `gauss` method my test gives an `output_raster` that is "shifted" by 1 pixel w.r.t. `expected_raster`.

That is why I have excluded `gauss` from the `parametrize` but it is not ideal, maybe there is something wrong with my test?

Reply:
@glostis in 1.1.5 we fixed a bug that could produce shifts in some cases. Are you up to date?

Also: would you be willing to leave gauss in, but mark it with xfail following the example in https://docs.pytest.org/en/latest/skipping.html#skip-xfail-with-parametrize ?

Reply:
I've branched off of https://github.com/mapbox/rasterio/commit/925718ea5cc61eaca46dcdb03739ecdde18f8e21 which was just post 1.1.5 release I believe, so unfortunately it does not seem that the fix solves the issue I've encountered in the test.

I've added gauss to the test with xfail as you have suggested in 1bcd5c906676d671fe1f2472a164692aae57c663, thanks for the tip!

Reply:
gauss xfails in the [CI tests](https://travis-ci.org/github/mapbox/rasterio/jobs/696487339#L2303)

Reply:
I only recently learned about xfail & parametrize. I like how it helps annotate and organize the tests.